### ISSUE TITLE
Error editing website

### ISSUE THREAD

**Issue by SamKacer** [mltony/nvda-browser-nav#22]
Hello, thanks for the fantastic add-on. I encountered an error just now. I have a website profile that I wanted to edit. I was on the website that the profile is for, so I used the NVDA + J dialogue to find the menu item to edit this website and hit enter. The dialogue failed to open.

I tried going via the NVDA settings menu instead. I went to BrowserNav quick jump settings, found the website in the list, and pressed edit. Same thing, the dialogue failed to open.

Checking the NVDA log, it seems to be caused by this error:

```
ERROR - unhandled exception (07:33:16.758) - MainThread (15032):
Traceback (most recent call last):
  File "C:\Users\samue\AppData\Roaming\nvda\addons\browsernav\globalPlugins\browserNav\quickJump.py", line 1808, in <lambda>
    lambda evt, site=site: editOrCreateSite(self, site=site),
  File "C:\Users\samue\AppData\Roaming\nvda\addons\browsernav\globalPlugins\browserNav\quickJump.py", line 1785, in editOrCreateSite
    entryDialog=EditSiteDialog(None, knownSites=knownSites, site=site, url=url, domain=domain, config=config)
  File "C:\Users\samue\AppData\Roaming\nvda\addons\browsernav\globalPlugins\browserNav\quickJump.py", line 2603, in __init__
    self.site.autoSpeakCategory if self.    site.autoSpeak else None
ValueError: <BookmarkCategory.QUICK_SPEAK: 9> is not in list
```

For more context, for this site,  I have autospeak enabled and a bunch of quick speak bookmarks.

- NVDA version: 2023.3.4
- BrowserNav version: 2.5
